src/apps/backend/modules/authentication/rest-api/authentication-controller.ts

```diff
@@ -21,15 +21,16 @@ export class AccessTokenController {
 
       if (username && password) {
         accessToken =
-          await AccessTokenService.createAccessTokenByUsernameAndPassword(
+          await AuthenticationService.createAccessTokenByUsernameAndPassword(
             password,
             username
           );
       } else if (phoneNumber && otpCode) {
-        accessToken = await AccessTokenService.createAccessTokenByPhoneNumber(
-          otpCode,
-          new PhoneNumber(phoneNumber.countryCode, phoneNumber.phoneNumber)
-        );
+        accessToken =
```

Comment:
Can you check why there is a difference in formatting? Please make sure there are no unwanted formatting changes.